nova_vm/src/engine/bytecode/bytecode_compiler.rs

```diff
@@ -48,7 +53,26 @@ pub(crate) enum NamedEvaluationParameter {
     ReferenceStack,
 }
 
-pub(crate) struct CompileContext<'agent, 'gc, 'scope> {
+pub(crate) struct JumpTarget {
```

Comment:
```suggestion
#[derive(Debug, Default)]
pub(crate) struct JumpTarget {
```

Reply:
Hmm, this is never debug printed so that'd be just a bit of unnecessary macro expansion, and the default implementation on this should never be used (it'd give lexical depth of 0 which may not be right).

I'll leave these out.